src/main/java/com/google/flourbot/execution/MacroExecutionModuleImplementation.java

```diff
@@ -66,15 +67,25 @@ public String execute(String userEmail, String message) throws IOException, Gene
         SheetAppendAction a = (SheetAppendAction) optionalMacro.get().getAction();
         String documentId = a.getSheetId();
         DriveClient cdc = new DriveClient();
-        CloudSheet cs = cdc.getCloudSheet(documentId);
-        cs.appendRow(values);
+        CloudSheet cloudSheet = cdc.getCloudSheet(documentId);
+        cloudSheet.appendRow(values);
+        break;
+      case SHEET_READ_ROW:
+        // TODO: after merge with master, move CloudSheet init to before the switch statement
+        CloudSheet cloudSheet = cloudDocClient.getCloudSheet(documentId);
+        List<String> sheetData = cloudSheet.readRow(2); //TODO: Replace hardcoded
```

Comment:
normally I'd say it is better to leave this change for later when we have the proper value from the parameter, but it's ok to merge to master now that you have it I guess.